.github/workflows/generate-data.yaml

```diff
@@ -40,45 +67,25 @@ jobs:
             bot=true
           fi
           echo "bot=$bot" >> "$GITHUB_OUTPUT"
-  build-evals:
-    needs: [check]
-    name: "Generate Eval Page Data"
-    runs-on: ubuntu-latest
-    continue-on-error: true
-    outputs:
-      ok: ${{ fromJSON(steps.status-check.outputs.ok) }}
-    env:
-      KEY: ${{ matrix.site.owner }}-${{ matrix.site.name }}
-    strategy:
-      matrix:
-        site: ${{ fromJSON(inputs.repos) }}
-    container:
-      image: ghcr.io/hubverse-org/hubpredevalsdata-docker:main
-      ports:
-        - 80
-      volumes:
-        - ${{ github.workspace }}:/project
-    steps:
       - id: id
-        name: "setup run variables"
+        name: "Setup Run Variables"
         run: |
-          if [[ -n "${{ matrix.site.build }}" ]]; then
-            echo "build=${{ matrix.site.build }}" >> $GITHUB_OUTPUT
-          else
-            echo "build=both" >> $GITHUB_OUTPUT
-          fi
-          owner="${{ matrix.site.owner }}"
-          name="${{ matrix.site.name }}"
+          owner="${{ env.OWNER }}"
+          name="${{ env.REPO }}"
+          key="${{ env.OWNER}}-${{ env.REPO }}"
+          echo "owner=$owner" >> $GITHUB_OUTPUT
+          echo "name=$name" >> $GITHUB_OUTPUT
+          echo "key=$key" >> $GITHUB_OUTPUT
           echo "repo=$owner/$name" >> $GITHUB_OUTPUT
-          echo "file=$owner-$name-predevals-data" >> $GITHUB_OUTPUT
+          echo "eval-art=$key-eval-data" >> $GITHUB_OUTPUT
+          echo "forecast-art=$key-forecast-data" >> $GITHUB_OUTPUT
```

Comment:
This step was duplicated in both build-evals and build-forecasts, so I moved them up to the setup step and generated most of the variables here.